### PR TITLE
add addUser.js test

### PR DESCRIPTION
**Pull request by 31453**
implements the addUser.js test for #1721

### CODE REVIEW

File: tests/addUser.t [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+system("cd ../viewer ; node addUser.js -c ../tests/config.test.ini -n testuser test2 test2 test2 --apionly");
+system("cd ../viewer ; node addUser.js -c ../tests/config.test.ini -n testuser test3 test3 test3 --email");
+system("cd ../viewer ; node addUser.js -c ../tests/config.test.ini -n testuser test4 test4 test4 --expression 'ip.src == 10.0.0.1'");
+system("cd ../viewer ; node addUser.js -c ../tests/config.test.ini -n testuser test5 test5 test5 --remove");
+system("cd ../viewer ; node addUser.js -c ../tests/config.test.ini -n testuser test6 test6 test6 --webauth");
+system("cd ../viewer ; node addUser.js -c ../tests/config.test.ini -n testuser test7 test7 test7 --packetSearch");
+
+my $users = viewerPost("/api/users", "");
+
+eq_or_diff($users->{recordsTotal}, 8, "Should have 9 users");
```
Comment (awick) [inline tests/addUser.t:22]: 8 or 9?

File: tests/addUser.t [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+ok($users->{data}->[0]->{createEnabled}, "Admin user");
+ok(!$users->{data}->[1]->{createEnabled}, "Not admin user");
+ok(!$users->{data}->[2]->{webEnabled}, "API only");
+ok($users->{data}->[3]->{emailSearch}, "Email Search");
+eq_or_diff($users->{data}->[4]->{expression}, "ip.src == 10.0.0.1");
+ok($users->{data}->[5]->{removeEnabled}, "Remove");
+ok($users->{data}->[6]->{headerAuthEnabled}, "Web auth");
+ok($users->{data}->[7]->{packetSearch}, "Packet search");
+# TODO not sure how to test the --webauthonly flag
```
Comment (awick) [inline tests/addUser.t:31]: you probably can't since it messes with the password, probably just make sure its there.

**Review by awick — CHANGES_REQUESTED** [review]

File: tests/addUser.t (hunk 1 above)
Comment (31453) [reply tests/addUser.t:22]: 😆 whoops, definitely 8! duh!

File: tests/addUser.t (hunk 2 above)
Comment (31453) [reply tests/addUser.t:31]: But the users api doesn't return the passStore property of a user 😕

Comment (31453) [reply tests/addUser.t]: fetched users directly from ES. DUH

**Pull request merged**